programs/clearing_house/src/controller/liquidation.rs

```diff
@@ -684,7 +698,7 @@ pub fn liquidate_borrow(
         )?;
     }
 
-    if liability_transfer_for_user >= liability_transfer_to_cover_margin_shortage {
+    if liability_transfer >= liability_transfer_to_cover_margin_shortage {
```

Comment:
shouldnt this be 
```liability_transfer
                .checked_sub(if_fee)
                .ok_or_else(math_error!())?```

Reply:
or `liability_transfer_to_cover_margin_shortage` should know about the `if_fee`?

Reply:
is there a test with a high `if_fee` where you check their margin is over the buffer?

Reply:
the `liability_transfer_to_cover_margin_shortage` assumes the `if_fee` is charged in addition to the `liability_transfer`

Reply:
I do have a test here: https://github.com/drift-labs/protocol-v2/pull/91/files#diff-6cc752f6206b4e6cd73442bb27b1ae22fd884efa4d736535d8eb7f1b6b5771a8R1125

it also checks that the margin ratio right about buffer